lib/directive/ng_model.dart

```diff
@@ -828,7 +828,7 @@ class ContentEditable extends InputTextLike {
 
   // The implementation is identical to InputTextLike but use innerHtml instead of value
   String get typedValue => (inputElement as dynamic).innerHtml;
-  void set typedValue(String value) {
-    (inputElement as dynamic).innerHtml = (value == null) ? '' : value;
+  void set typedValue(value) {
```

Comment:
I don't know when value is not a String.
The important thing is keeping this method signature consistent with the base class which does not restrict value to a String.
I'm equally happy with changing this code to 
 void set typedValue(value) {
  assert(value is String);
  (inputElement as dynamic).innerHtml = (value == null) ? '' : value;
}
Let me know which you'd prefer.